.github/workflows/release.yml

```
@@ -46,7 +46,7 @@ jobs:
           else
             opt_prerelease="--prerelease"
           fi
-          gh release create --target main \
+          gh release create --target HEAD \
                             --title "Application Insights Java $VERSION$opt_ga" \
```

I think `HEAD` is the default

```suggestion
          gh release create --title "Application Insights Java $VERSION$opt_ga" \
```